source/dofs/dof_handler.cc

```diff
@@ -1203,36 +1203,9 @@ namespace internal
           // if so skip it (consequently, we have to save and later
```

Comment:
This comment is out of date with this change - otherwise 👍

Reply:
Thanks, let me fix it now.